src/main/steps/applicant1/check-your-answers/content.ts

```diff
@@ -51,28 +51,60 @@ const getHelpWithFeesMoreDetailsContent = (applicant1HelpPayingNeeded, isDivorce
   return moreDetailsComponent({ text, title });
 };
 
-const getOtherCourtCasesMoreDetailsContent = (isDivorce: boolean) => {
+const getCyHelpWithFeesMoreDetailsContent = (applicant1HelpPayingNeeded, isDivorce, checkTheirAnswersPartner) => {
+  const title = 'Darganfyddwch fwy am help i dalu ffioedd';
+  const text = `Mae'r cais ${isDivorce ? 'am ysgariad' : "i ddod â'ch partneriaeth sifil i ben"} yn costio ${getFee(
+    config.get('fees.applicationFee')
+  )}.
+  Ni ofynnir i chi dalu'r ffi. Gofynnir i'ch ${checkTheirAnswersPartner} dalu. ${
+    applicant1HelpPayingNeeded === YesOrNo.YES
+      ? "Maent wedi dweud bod angen help arnynt i dalu'r ffi. Gallant ond gael help i dalu ffioedd os byddwch chi yn gwneud cais hefyd. Dyna pam y gofynnwyd ichi a oedd angen help arnoch i dalu'r ffi."
+      : "Maent wedi dweud nad oes angen help arnynt i dalu'r ffi."
+  }`;
+
+  return moreDetailsComponent({ text, title });
+};
+
+const getEnOtherCourtCasesMoreDetailsContent = (isDivorce: boolean) => {
   const title = 'Find out more about other court proceedings';
   const text = `The court only needs to know about court proceedings relating to your ${
     isDivorce ? 'marriage' : 'civil partnership'
   }, property or children. It does not need to know about other court proceedings.`;
   return moreDetailsComponent({ text, title });
 };
 
+const getCyOtherCourtCasesMoreDetailsContent = (isDivorce: boolean) => {
+  const title = 'Rhagor o wybodaeth am achosion llys eraill';
+  const text = `Nid oes ond angen i'r llys wybod am achosion llys sy'n ymwneud â'ch ${
+    isDivorce ? 'priodas' : 'partneriaeth sifil'
+  } eiddo neu blant. Nid oes angen iddo wybod am achosion llys eraill.`;
+  return moreDetailsComponent({ text, title });
+};
+
 const cannotUploadDocumentList = (
   isDivorce: boolean,
+  isWelsh: boolean,
```

Comment:
i think we need to stitch with one variable as in other places, we have isEnglish. its not a big deal but itll be nice to keep it consistent.

Reply:
I'll change it to isEnglish